packages/c/sshnpd/include/sshnpd/run_srv_process.h

```diff
@@ -1,10 +1,11 @@
 #ifndef RUN_SRV_H
 #define RUN_SRV_H
 
-#include "sshnpd/params.h"
 #include <atclient/cjson.h>
+#include <stdbool.h>
+#include <stdint.h>
 
-int run_srv_process(sshnpd_params *params, cJSON *host, cJSON *port, bool npt, cJSON *requested_host,
-                    cJSON *requested_port, bool authenticate_to_rvd, char *rvd_auth_string, bool encrypt_rvd_traffic,
-                    bool multi, unsigned char *session_aes_key_encrypted, unsigned char *session_iv_encrypted);
+int run_srv_process(const char *srvd_host, uint16_t srvd_port, const char *requested_host, uint16_t requested_port,
+                    bool authenticate_to_rvd, char *rvd_auth_string, bool encrypt_rvd_traffic, bool multi,
+                    unsigned char *session_aes_key_encrypted, unsigned char *session_iv_encrypted);
```

Comment:
removed `sshnpd_params *params`, `bool npt` which were redundant

changed all `cJSON *` objects to be `char*` and `unint16_t` which are the primitive types used by srv.

This way we are not passing around cJSON referencees which may result in potential memory issues